openmc/tallies.py

```diff
@@ -452,9 +449,13 @@ def add_score(self, score):
                   'not a string'.format(score, self.id)
             raise ValueError(msg)
 
-        # If the score is already in the Tally, don't add it again
+        # If the score is already in the Tally, raise an error
         if score in self.scores:
-            return
+            msg = 'Unable to add a duplicate score {0} to Tally ID="{1}" ' \
```

Comment:
@paulromano if you are fine with the remainder of this PR then I believe it
is ready to merge.
On Dec 26, 2015 12:32 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/tallies.py
> <https://github.com/mit-crpg/openmc/pull/534#discussion_r48448540>:
>
> >          if score in self.scores:
> > -            return
> > +            msg = 'Unable to add a duplicate score {0} to Tally ID="{1}" ' \
>
> I can volunteer to make a fix for input_xml.F90.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/534/files#r48448540>.
>